openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.item.dropped.json

```diff
@@ -0,0 +1,48 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Completed",
```

Comment:
Why `Completed`?

Reply:
because i think this is the most suitable one among the available `action` options for `AssessmentItemEvent` .
source: https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html#assessmentItemEvent